source/administration/telemetry.rst

```diff
@@ -46,8 +46,12 @@ Mattermost error and diagnostic data is collected for the following purposes:
 .. note:: 
 Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. To opt out, you can disable the feature in **System Console > Environment > Logging** (or **System Console > General > Logging > Enable Error and Diagnostics Reporting** in versions prior to 5.12).
 
+Reporting Frequency
+  - Every 10 minutes for the first hour after ther server is booted up for the first time
+  - Every hour thereafter for the first 12 hours
+  - At the 24 hour mark and every 24 hours after
 
-The following data is sent once every 24 hours:
+The following data is sent:
```

Comment:
Currently, there are two subsections for "Error and Diagnostics Reporting Feature" 
 - [The following data is sent once every 24 hours](https://docs.mattermost.com/administration/telemetry.html#the-following-data-is-sent-once-every-24-hours)
 - [The following information is sent when the specified event occurs](https://docs.mattermost.com/administration/telemetry.html#the-following-information-is-sent-when-the-specified-event-occurs)

Does the new reporting frequency apply to the first of these subsections only? If so, let's make that clear

Reply:
OK made a couple more edits. Hopefully clearer now, but had to re-title some sections for which there isn't a clear title. Let me know if it works better